src/utils/DockerUtil.js

```diff
@@ -212,6 +213,7 @@ export default {
   run (name, repository, tag) {
     tag = tag || 'latest';
     let imageName = repository + ':' + tag;
+    name = ContainerStore.generateName(name);
```

Comment:
Disregard the previous comment, it would be best to:

- Call `ContainerStore.generateName` in the `NewContainerPull` component instead of here.